app/models/api/v3/download_version.rb

```diff
@@ -1,6 +1,12 @@
 module Api
   module V3
     class DownloadVersion < BaseModel
+      def self.current_version_symbol(context)
+        current_version = where(is_current: true, context_id: context.id).
+          order('created_at DESC').
```

Comment:
![image](https://user-images.githubusercontent.com/973964/34676093-f3d45140-f48b-11e7-97f2-23252302d259.png)

I'm getting v1.0 for Brazil soy. We should probably add a 2nd "order by" clause, per symbol maybe.

Reply:
I think that's because there was a bug in the SchemaRevamp.copy logic, where `created_at` was set to `NOW()` at the time of copying rather than copying the existing `created_at` value. That's also fixed in this PR, but requires running the copy task again.

Reply:
If we think `created_at` is not reliable, what would be good I guess is a constraint to allow only one row to have `is_current` set.

Reply:
that would make sense, yes.

Reply:
I added the constraint to our database, I'll also recommend adding it to main db.